src/components/FeedbackPage/FeedbackPage.scss

```diff
@@ -0,0 +1,31 @@
+@import '~style/variables';
+
+.feedbackWrapper {
+  @include container;
+
+  padding: 48px 0;
```

Comment:
This overrides the left/right padding of the container so it gets smooshed up against the sides
![image](https://user-images.githubusercontent.com/4443482/32812813-06c57dd0-c9fb-11e7-95f0-938d5f571888.png)

But is fixed by having a separate container and wrapper, see below.